_release/cacophony-config-import.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Import old config to new config
+After=multi-user.target network.target
+ConditionPathExists=!/etc/cacophony/config.toml
+
+[Service]
+ExecStart=/usr/bin/cacophony-config-import
+Type=oneshot
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I'm pretty sure you need to make this run earlier in the boot process to ensure it starts before any of our other code.

I *think* you want something like `WantedBy=basic.target` here and `After=basic.target` in `[Unit]` but you'll want to check this as I'm going off memory.

Reply:
Yep. That's much better than just letting the our other services fail and then restart :)